### PR TITLE
Remove obsolete dnceng mirroring entries (2)

### PR DESCRIPTION
**Pull request by pavelsavara**
I removed subscriptions which
- don't map to existing branches in GitHub
- repo in github doesn't exist
- repo in azdo doesn't exist
- repo in azdo is archived

Few smaller tweaks

Continuation of https://github.com/dotnet/versions/pull/973/

This is just based on my exploration, not authoritative. Let's bring in somebody who could validate.

### CODE REVIEW

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -553,23 +493,6 @@
         }
       }
     },
-    // Automate merging runtime release/7.0 branch into release/7.0-staging
```
Comment (pavelsavara) [inline Maestro/subscriptions.json:556]: release/7.0 is out of support

File: Maestro/subscriptions.json [hunk 2]
```diff
@@ -1474,40 +1396,6 @@
           "CommitsToMirror": "<trigger-commitlist>"
         }
       }
-    },
-    // Automate opening PRs to merge winforms-designer release/16.10p4 to release/16.11p1
```
Comment (pavelsavara) [inline Maestro/subscriptions.json:1478]: azdo repository is disabled
https://dev.azure.com/dnceng/internal/_git/dotnet-winforms-designer

File: Maestro/subscriptions.json [hunk 3]
```diff
@@ -96,12 +96,6 @@
       "vsoProject": "DevDiv",
       "buildDefinitionId": 7644
     },
-    // A build definition that will update the dependencies of the dotnet/toolset repo
```
Comment (pavelsavara) [inline Maestro/subscriptions.json:99]: https://github.com/dotnet/toolset/ is archived

File: Maestro/subscriptions.json [hunk 4]
```diff
@@ -343,7 +326,7 @@
         "https://github.com/dotnet/extensions/blob/dev/**/*",
         "https://github.com/dotnet/extensions/blob/main/**/*",
         "https://github.com/dotnet/extensions/blob/release/**/*",
-        "https://github.com/dotnet/format/blob/main/**/*",
+        "https://github.com/dotnet/format/blob/archive/**/*",
```
[the remark this answers is not in the document]
Comment (mmitche) [reply Maestro/subscriptions.json:329]: You can remove the `archive` entry.

File: Maestro/subscriptions.json [hunk 5]
```diff
@@ -1183,7 +1106,7 @@
     // Merge mirrored branches (to internal/) in dnceng Azure Devops
     {
       "triggerPaths": [
-        "https://github.com/dotnet/aspnetcore/blob/release/2/**/*",
+        "https://github.com/dotnet/aspnetcore/blob/release/2.1/**/*",
```
[the remark this answers is not in the document]
Comment (mmitche) [reply Maestro/subscriptions.json:1109]: This matches 2*. There used to be a 2.2 branch is my guess. @wtgodbe?

File: Maestro/subscriptions.json (hunk 2 above)
Comment (mmitche) [reply Maestro/subscriptions.json:1478]: This should stay. This repo builds from GH.

**Review by mmitche — CHANGES_REQUESTED** [review]
Few tweaks, but otherwise looks good.

File: Maestro/subscriptions.json (hunk 5 above)
[the remark this answers is not in the document]
Comment (pavelsavara) [reply Maestro/subscriptions.json:1108]: this was `2/` it would not match `2.1` nor `2.2`. Right ?

[the remark this answers is not in the document]
Comment (mmitche) [reply Maestro/subscriptions.json:1107]: It removes `/**/*` before comparison.

[the remark this answers is not in the document]
Comment (wtgodbe) [reply Maestro/subscriptions.json:1107]: > This matches 2*. There used to be a 2.2 branch is my guess. @wtgodbe?

Yeah, that was it. We can just make it 2.1 now.

**Review by mmitche — APPROVED** [review]

**Pull request merged**